browser/brave_browser_process_impl.cc

```diff
@@ -57,6 +57,10 @@
 
 #if BUILDFLAG(ENABLE_TOR)
 #include "brave/browser/extensions/brave_tor_client_updater.h"
+#include "brave/browser/ui/brave_browser_command_controller.h"
```

Comment:
even though these are only used when tor is enabled, I still think it's a bad idea to put them in this guard because in the future other things may use these same header files and people may not move them out of the ENABLE_TOR block when that happens. I think only tor header files should go in here.

Reply:
In nightly, tor process is gone. but not in stable version.